app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class RentsController < ApiController
+  def index
+    render json: Rent.all.page(params[:page])
+  end
+
+  def create
+    rent = Rent.new(rent_params)
+    if rent.save
+      render json: rent
+    elsif rent.book_id.nil? || rent.user_id.nil? || rent.to.nil? || rent.from.nil?
+      render json: { status: 'error',
+                     code: 400,
+                     message: "Can't save a rent, one or more parameters of the request are null" }
+    else
+      render json: { status: 'error', code: 500, message: "Can't save a rent" }
```

Comment:
Same on the success status code, also we shouldn't be returning a 500 from the controller like this, if something fails along the stack the exception will propagate up.